code/online_challenges/src/coderbyte/first_reverse/firstReverse.cpp

```diff
@@ -0,0 +1,20 @@
+#include <iostream>
+#include <algorithm>
+using namespace std;
+
+string reverseIt(char* str)
+{
+	string st(str);
+	reverse(st.begin(), st.end());
+	return st;
```

Comment:
You could also do (Without having to call the reverse function and including algorithm):

```C++
return string{st.rbegin(), st.rend()};
```

Reply:
but i used a standard cpp function @arnavb